### PR TITLE
Intake Subsystem Skeleton

### LINKED ISSUE #7: IntakeSubsystem
- [x] Create a skeleton intake subsystem
- [x] Stop when beam-break sensor is triggered

### PR DESCRIPTION
**Pull request by having11**
Closes #7

### CODE REVIEW

File: src/main/include/Constants.h [hunk 1]
```diff
@@ -170,9 +170,18 @@ constexpr uint8_t kLedAddress = 23;
 // Motor IDs
 namespace CANSparkMaxConstants {
 constexpr int kIntakeSpinnyBoyID = 20;
+constexpr int kVectorSpinnyBoyID = 21;
```
Comment (nobody5050) [inline src/main/include/Constants.h:173]: should be kIntakeSpinnyBo**i** and kVectorSpinnyBo**i** ;)

**Review by nobody5050 — DISMISSED** [review]

File: src/main/cpp/RobotContainer.cpp [hunk 2]
```diff
@@ -59,31 +55,30 @@ RobotContainer::RobotContainer() {
...
       .WhileTrue(new frc2::RunCommand([this] { m_drive.SetX(); }, {&m_drive}));
 
-#ifdef TEST_SWERVE_BOT
-    // m_wrist->SetDefaultCommand(
-    //     RotateWrist(m_wrist.get(), [this] {
-    //         return -m_operatorController.GetRightY();
-    //     }));
+  frc2::JoystickButton(&m_driverController,
+                      frc::XboxController::Button::kX)
+      .OnTrue(m_leds.ShowFromState([this] { return m_stateManager.getState(); }));
```
[the remark this answers is not in the document]
Comment (having11) [reply src/main/cpp/RobotContainer.cpp:70]: 🔥

File: src/main/include/Constants.h (hunk 1 above)
Comment (having11) [reply src/main/include/Constants.h:173]: 100%

**Review by nobody5050 — APPROVED** [review]
Much better

**Pull request merged**